tests/e2e/clinicianPageFunctional.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable linebreak-style */
+require('../../utilities/seleniumKeepAlive');
+
+module.exports = {
+  '@tags': ['clinician'],
+  'Clinician User Logs in with Existing Credentials'(browser) {
+    const loginPage = browser.page.loginPage();
+    const clinicianUsername = browser.globals.clinicianUsername;
+    const clinicianPassword = browser.globals.clinicianPassword;
+    loginPage.loadPage();
+    loginPage.userLogin(clinicianUsername, clinicianPassword);
+  },
+  'Clinic workspace selection'(browser) {
```

Comment:
This is test setup. Logging into Tidepool. We have it pulled out since it's repeated over and over again to start tests. Potentially we could make a function specifically devoted to "test setup" so it's more clear, but that can be another PR.